examples/ingestion-pipeline/main.py

```diff
@@ -0,0 +1,112 @@
+"""
+  Copyright 2020 Google LLC
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+ 
+    https://www.apache.org/licenses/LICENSE-2.0
+ 
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ """
+
+from __future__ import absolute_import
+import argparse
+import logging
+from past.builtins import unicode
+import apache_beam as beam
+from apache_beam.io import WriteToText
+from apache_beam.options.pipeline_options import PipelineOptions
+from apache_beam.options.pipeline_options import SetupOptions
+import firebase_admin
+from firebase_admin import credentials
+from firebase_admin import firestore
+import random
+import flickrapi
+from providers import imageProviderFlickr
+from datetime import datetime
+
+
+JOB_NAME = 'image-ingestion-job: ' + str(datetime.now()) #Global name for the job that will run
+
+def initialize_database(): 
+    if not firebase_admin._apps:
+        firebase_admin.initialize_app(credentials.ApplicationDefault(), {
+        'projectId': 'step-project-ellispis',
+        })
+    return firestore.client()
+
+class UploadToDatabase(beam.DoFn):
+    def setup(self):
+        self.db = initialize_database()
+    def process(self, element):
+        location=firestore.GeoPoint(float(element.location[0]), float(element.location[1]))
+        doc_ref = self.db.collection(u'imagesDemoTal02').document()
+        sub_doc_ref= doc_ref.collection(u'pipelineRun').document()
```

Comment:
make this a constant defined at the top of the file

Reply:
changed collection and sub collection to global variables